MaxText/llama_or_mistral_ckpt.py

```diff
@@ -30,25 +32,30 @@
 # pylint: disable=g-line-too-long
 import argparse
 import pathlib
+import os
+import sys
+import gc
+import re
+import logging
+from dataclasses import dataclass
 
-import numpy as np
+os.environ["JAX_PLATFORMS"] = "cpu"
 
-import checkpointing
+# pyling: disable
```

Comment:
Does this do anything? I thought its pylint not pyling

Reply:
:D thanks